packages/conveyer/src/Conveyer.ts

```diff
@@ -482,9 +496,12 @@ class Conveyer extends Component<ConveyerEvents> {
     }
     return scrollPos;
   }
-  private _checkNestedMove(e: any) {
+  private _isMixedWheel(nativeEvent: any) {
+    return nativeEvent?.type === "wheel" && nativeEvent?.deltaX && nativeEvent?.deltaY;
```

Comment:
How about adding `nativeEvent &&` before all of these?

Reply:
It seems that adding `!!nativeEvent` can improve readability.